test/unit/src/common/actions/t_register-name.js

```diff
@@ -16,6 +16,7 @@ import {
 } from '../../../../../src/common/actions/snap-builds';
 import { conf } from '../../../../../src/common/helpers/config';
 import { makeLocalForageStub } from '../../../../helpers';
+import callApi from '../../../../../src/common/middleware/call-api';
```

Comment:
Having now tasted a world where tests live beside the code they test, I find this path line noise even more annoying that I did previously :(